docs/setup_guides/azure.rst

```diff
@@ -0,0 +1,249 @@
+    --min-count 1 \
+    --max-count 10
+
+You can also update an existing Kubernetes service to add autoscaling:
+
+.. code-block:: bash
+
+  az aks update \
+  --resource-group $RESOURCE_GROUP_NAME \
+  --name $AKS_RESOURCE_NAME \
+  --enable-cluster-autoscaler \
+  --min-count 1 \
+  --max-count 3
+
+More information on autoscaling with Azure AKS is available here:
+https://docs.microsoft.com/en-gb/azure/aks/cluster-autoscaler.
+
+
+Step Two: Customise cluster
+===========================
+
+With a working Kubernetes service now built we can customise it in readiness for installing Pangeo
+on the cluster. At its most basic, this means installing helm and tiller, but other
+customisations (such as authentication) can also be added at this stage.
+The customisations need to be performed using the Azure CLI. If you don't have the Azure CLI available,
+you can either:
+
+* follow the steps at the link above to install the Azure CLI locally, or
+* use the cloud shell built into the web interface
+  (click the ``>_`` logo at the right of the blue bar at the top of the web interface).
+  The cloud shell includes the Azure CLI and a basic implementation of Visual Studio Code editor.
+
+Kubernetes credentials
+~~~~~~~~~~~~~~~~~~~~~~
+
+Before we can progress we need to acquire kubernetes credentials for our newly-created
+AKS resource:
+
+.. code-block:: bash
+
+  az aks get-credentials -g $RESOURCE_GROUP_NAME -n $AKS_RESOURCE_NAME --overwrite-existing
+
+
+You will need to provide the name of the AKS resource that you just created (as ``$AKS_RESOURCE_NAME``)
+and the group within which the resource was created (as ``$RESOURCE_GROUP_NAME``).
+
+
+Helm and tiller
+~~~~~~~~~~~~~~~
+
+Installing helm and tiller allows us to customise our Kubernetes service by applying
+helm charts to it. We need to ensure that helm and tiller will work correctly with
+RBAC, which is enabled by default on Azure Kubernetes services.
+
+.. code-block:: bash
+
+  kubectl apply -f helm_rbac.yaml
+  helm init --upgrade --service-account tiller --wait
+
+The contents of ``helm_rbac.yaml`` are as follows:
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: tiller
+    namespace: kube-system
+  ---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: tiller
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-admin
+  subjects:
+    - kind: ServiceAccount
+      name: tiller
+      namespace: kube-system
+
+
+Step 3: Install Pangeo
+======================
+
+Now we can move onto installing Pangeo on our Kubernetes service. This can be done
+as follows:
+
+.. code-block:: bash
+
+  helm repo add pangeo https://pangeo-data.github.io/helm-chart/
+  helm repo update
+  helm upgrade --install --namespace pangeo pangeo pangeo/pangeo -f pangeo.yaml
+
+The contents of the ``pangeo.yaml`` file came from the Zero to Jupyterhub guide.
```

Comment:
I feel like we are skipping over the ` pangeo.yaml` file a bit here. We should probably provide an example.

Reply:
I feel like I meant to add more content here, but didn't...